inc/namespace.php

```diff
@@ -141,3 +164,22 @@ function get_endpoint(): string {
 
 	return $endpoint;
 }
+
+/**
+ * Get the authentication token.
+ *
+ * @return string|null
+ */
+function get_auth_token() : ?string {
+
+	$token = defined( 'AMF_WORDPRESS_TOKEN' ) ? AMF_WORDPRESS_TOKEN : get_option( TOKEN_SETTING, null );
+
+	/**
+	 * Filters the REST API authentication token.
+	 *
+	 * @param string $token The authentication token for the REST API authorization header.
+	 */
+	$token = apply_filters( 'amf_wordpress_token', $token );
+
+	return ! empty( $token ) ? base64_encode( $token ) : null;
```

Comment:
Why `null`? If we make this an empty string, we can be type-safe by declaring `string`. Does that not work anywhere?